### PR TITLE
Markup for chatroom page

### PR DESCRIPTION
**Pull request by Gatts1**
-Solve #3 
-Create 2 new issues to complete this.
#18 #19

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  display: grid;
+  grid-template:12% 1fr/ 20% 1fr;
+  height: 100vh;
+  background-color: #656463;
+  grid-template-areas:
```
Comment (Linzeur) [inline public/style.css]: Can you explain me what that property does?

File: public/style.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+'header main'
+'app_name head_channel'
+}
+
+/* Variables */
+header { grid-area: header;}
+main { grid-area: main; }
+.head_channel { grid-area: head_channel;}
```
Comment (diegotc86) [inline public/style.css]: This class name is not included on the HTML page. Where it should be included?

File: public/style.css (hunk 1 above)
Comment (paulotijero) [reply public/style.css]: This is another format that creates the grid, could be declared `grid-template: rows / columns`, them in `grid-template-areas` you can paint the grit as if was a matrix like this:
```CSS
body {
  height: 100vh;
  display: grid;
  grid-template: 12% 1fr / 20% 1fr;
  grid-template-areas:
    'pos1-1 pos1-2'
    'pos2-1 pos2-2'
}
```
Then you need to create the classes like this:
```CSS
.pos1-1 {  grid-area: pos1-1;  background-color: red; }
.pos1-2 {  grid-area: pos1-2;  background-color: blue; }
.pos2-1 {  grid-area: pos2-1;  background-color: yellow; }
.pos2-2 {  grid-area: pos2-2;  background-color: orange; }
```
So, to make it work have adding the classes in your `html`
```HTML
<body>
  <section class="pos1-1"></section>
  <section class="pos1-2"></section>
  <section class="pos2-1"></section>
  <section class="pos2-2"></section>
</body>
```
You will get a result like this:
grid

File: public/style.css (hunk 2 above)
Comment (paulotijero) [reply public/style.css]: You right :thinking:, I will try solving this.

File: public/style.css (hunk 1 above)
Comment (Linzeur) [reply public/style.css]: Oh, I got it, it creates a parent class in a manner of speaking to each section.

**Pull request merged**